services/pane.test.js

```diff
@@ -196,6 +198,42 @@ describe(dirname, function () {
 
         fn().then(expectScheduledPane);
       });
+
+      it('adds valid message if there are no warnings', function () {
+        getState.returns(Promise.resolve({}));
+        lib.close();
+
+        function expectRegularPane() {
+          expect(document.querySelector('.pane-header').innerHTML).to.equal('Schedule Publish');
+          expect(document.querySelector('.pane-inner .publish-valid').innerHTML).to.equal('valid');
+        }
+
+        fn().then(expectRegularPane);
+      });
+
+      it('adds warning message if warnings are passed in', function () {
+        getState.returns(Promise.resolve({}));
+        lib.close();
+
+        function expectRegularPane() {
+          expect(document.querySelector('.pane-header').innerHTML).to.equal('Schedule Publish');
+          expect(document.querySelector('.pane-inner .publish-valid')).to.equal(null);
```

Comment:
Still spinning up on tests. For Chai it looks like this could also end with `('.pane-inner .publish-valid')).to.not.exist` which looks for `null` and `undefined`. Do we care?